lib/tagged-type.d.ts

```diff
@@ -23,3 +23,8 @@ export declare class TaggedTypeFormat {
 }
 export declare const LONG_MIN: bigint;
 export declare const LONG_MAX: bigint;
+export declare class TaggedTypeEncoded {
+    #private;
```

Comment:
what does this do? Can't find docs on this here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/Private_class_fields

Reply:
I dunno! that's some weird typescript magic happening right there in the build files.

Reply:
oh lol that's the build file 😵‍💫 

got to move that!